src/components/tabController/apis/TabBar.api.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "TabController.TabBar",
+  "category": "layoutsAndTemplates",
+  "description": "TabController's TabBar component",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/TabControllerScreen/index.tsx",
+  "images": [],
```

Comment:
We decided to remove empty keys

Reply:
I kept the images so it'll remind me to add ones.. cause technically we want all components to have images/gifs